javascript/javascript1/week2/flight-booking.js

```diff
@@ -0,0 +1,17 @@
+// Flight booking fullname function and Formal fullname
+function getFullname(firstname, surname, useFormalName, gender) {
+  if (useFormalName) {
+    if (gender === "female") {
+      return "Queen " + firstname + " " + surname;
```

Comment:
[Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) might look cleaner here
```suggestion
      return `Queen ${firstName} ${surname}`;
```